docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG PYTHON_VERSION=3.13
+ARG EM_VERSION=4.0.1
+ARG EM_NODE_VERSION=20.18.0_64bit
+ARG ARM_GNU_TOOLCHAIN_VERSION=14.2.rel1
+ARG ARM_GNU_TOOLCHAIN_PLATFORM=x86_64-arm-none-eabi
+ARG PEBBLE_BOARD_LIST="snowy_bb2 spalding_bb2 silk_bb2 asterix_evt1"
+
+FROM python:${PYTHON_VERSION} AS base
+RUN apt update && apt install -y gcc-multilib gettext clang libusb-1.0-0-dev
+
+FROM base AS arm-toolchain
+ARG ARM_GNU_TOOLCHAIN_VERSION
+ARG ARM_GNU_TOOLCHAIN_PLATFORM
+RUN curl -L "https://developer.arm.com/-/media/Files/downloads/gnu/${ARM_GNU_TOOLCHAIN_VERSION}/binrel/arm-gnu-toolchain-${ARM_GNU_TOOLCHAIN_VERSION}-${ARM_GNU_TOOLCHAIN_PLATFORM}.tar.xz" -o arm-gnu-toolchain.tar.xz && \
+    tar xf arm-gnu-toolchain.tar.xz -C /opt && \
+    mv /opt/arm-gnu-toolchain-${ARM_GNU_TOOLCHAIN_VERSION}-${ARM_GNU_TOOLCHAIN_PLATFORM} /opt/arm-gnu-toolchain
+
+FROM base AS emsdk
+ARG EM_VERSION
+RUN git clone https://github.com/emscripten-core/emsdk.git /emsdk --depth 1 && \
```

Comment:
For reproducibility, would suggest pinning to a specific commit. And maybe even just doing curl/wget of a tarball rather than cloning via Git. Unless either of these build steps depend on it being a Git clone.

Reply:
Is `git clone --depth 1 --branch ${EM_VERSION}` sufficient enough?

Reply:
Added on newest commit, can you take a look again please?